modules/go-deleting-argocd-instance.adoc

```diff
@@ -14,19 +14,19 @@ Delete the Argo CD instances added to the namespace of the GitOps Operator.
 
 [source,terminal]
 ----
-$ oc delete gitopsservice cluster -n openshift-gitops
+$ oc delete gitopsservice cluster
 ----
 
 [NOTE]
 ====
 You cannot delete an Argo CD cluster from the web console UI.
 ====
 
-After the command runs successfully all the Argo CD instances will be deleted from the `openshift-gitops` namespace.
+After the above command runs successfully the default GitOps Service and its respective Argo CD instance will be deleted from the `openshift-gitops` namespace.
```

Comment:
We don't use "above" to indicate a relative location in our documents (see https://www.ibm.com/docs/en/ibm-style?topic=word-usage#above if you're interested). Instead, maybe revise to the following: 

After the previously command runs successfully, the default GitOps Service and its respective Argo CD instance will be deleted from the `openshift-gitops` namespace.